samples/SpanUsage/SpanUsage.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27604.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SpanUsage", "SpanUsage\SpanUsage\SpanUsage.csproj", "{A99829A8-F185-4044-904F-211077DD11B8}"
```

Comment:
Is it really a sample?

Reply:
> TODO: 1. Add to corefxlab.sln

It could be considered a sample code analyzer for span. I wanted to start it there, and move it to the solution.  Would you be fine with that?

Reply:
@terrajobst is working on a package for many .NET analyzers. I think this analyzer might be a good fit for such package.

Having said that, I am totally fine with adding it to this samples project short term.